common/decklist.cpp

```diff
@@ -579,8 +579,6 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         cardName.replace(rx, "'");
         rx.setPattern("Æ");
         cardName.replace(rx, "AE");
```

Comment:
I suppose we need to know what the data source (mtgjson) plans to do about this first.